### ISSUE TITLE
DEvice or resource busy

### ISSUE THREAD

**Issue by manodupont** [bob-california/opensearch-keycloak#1]
HI, i know this has not been maintained regularly but Im giving it a shot as I really would like to make this work.

The error : 
```
[manuel@ip-172-31-15-163 opensearch1]$ dl os01
opensearch1-os01-1  | Disabling execution of install_demo_configuration.sh for OpenSearch Security Plugin
opensearch1-os01-1  | Enabling OpenSearch Security Plugin
opensearch1-os01-1  | sed: cannot rename /usr/share/opensearch/config/sedNcg44Z: Device or resource busy
```

The docker-compose is the same as you have.

How did you make this work ?

I have been checking around all the users permissions etc, I just can;t seem to find the answer to that

If you have any idea, that would be very nice.

Thanks

**Comment by KeksTheAllmighty** [comment]
recieving the same error, any progress on getting keycloak work with opensearch @manodupont ?

**Comment by flavienbwk** [comment]
This looks like fixed in [`1.2.4`](https://github.com/opensearch-project/opensearch-build/pull/1130).

**Comment by cht42** [comment]
I upgraded the version of opensearch to 1.2.4, it should be fixed
